lib/team_stats.rb

```diff
@@ -1,8 +1,125 @@
 class TeamStats
 
-  attr_reader :game_teams, :teams
-  def initialize(game_teams, teams)
+  attr_reader :game_teams, :teams, :games
+  def initialize(game_teams, teams, games)
     @game_teams = game_teams
     @teams = teams
+    @games = games
+  end
+
+  def team_info(id)
+    team = @teams.find{|team| team.team_id == id}
+    team_info = {}
+    team_info[:team_id] = team.team_id
+    team_info[:franchise_id] = team.franchise_id
+    team_info[:abbreviation] = team.abbreviation
+    team_info[:team_name] = team.team_name
+    team_info[:link] = team.link
+    team_info
+  end
+
+  def season_from_game(game_id, team_id)
+    game_team = @game_teams.find do |game_team|
+      game_team.game_id == game_id && game_team.team_id == team_id
+    end
+    game_season = {}
+    game = @games.find{|game| game.game_id == game_id }
+    game_season[game.season] = game_team
+    game_season
+  end
+
+  def games_played_in(team_id)
+    @game_teams.select{|game| game.team_id == team_id}
+  end
+
+  def average_win_percentage(team_id)
```

Comment:
This look very similar to one of my methods, perhaps something we can create a module for.